modules/ROOT/pages/rest-sdk/edits/project-setup.adoc

```diff
@@ -0,0 +1,62 @@
+= Setup and Installation
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+
+To get started with REST SDK, ensure you meet the prerequisites and then follow the
+steps in this section to install the REST SDK plugin.
+
+== Prerequisites
+
+Before you start using REST SDK, verify that you have the tools needed:
+
+* Java 8 SDK
+* Apache Maven (3.3.9 or newer recommended)
+
+== Install the REST SDK Plugin
+
+Install the https://repository-master.mulesoft.org/nexus/service/local/repositories/releases/content/com/mulesoft/connectors/rest-connect-cli/1.0.0-RC2/rest-connect-cli-1.0.0-RC2-jar-with-dependencies.jar[REST SDK plugin] to use REST SDK. The plugin is a distribution
+of one or more related mojos (Maven plain Old Java Objects). Each mojo is an
```

Comment:
```suggestion
of one or more related _mojos_ (Maven plain Old Java Objects). Each mojo is an
```
HOWEVER...I looked this up, and I see that it is often all lowercase...but it makes more sense to me grammatically to present this as MOJO (as some sources do), because it is an initialism for Maven Old Java Objects. Let me know what you decide and I will add it to our Term Presentation List.

Reply:
Interesting as I believe it was actually _mojos_ but I changed it to mojos. But it looks like since it is an initialism, we should capitalize it as MOJO. Thank you for adding it to the Term Presentation List.